auth/types/pulp_test.go

```diff
@@ -0,0 +1,20 @@
+package types
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"gotest.tools/assert"
```

Comment:
Let's use `testify` since we are using it for other test cases as well.

Reply:
updated

Reply:
@jdkanani made changes as requested.